flaskServer/restEndpoint.py

```diff
@@ -0,0 +1,82 @@
+from flask import Flask, render_template, url_for, request, redirect
+from flask_restful import Resource, Api, reqparse
+from werkzeug import secure_filename
+from flask_cors import CORS, cross_origin
+import os
+from eventClass.eventHandler import EventHandler
+from eventClass.aeroCubeEvent import ImageEvent, ResultEvent, AeroCubeEvent
+from eventClass.aeroCubeSignal import *
+from eventClass.bundle import Bundle
+from tcpService.tcpClient import TcpClient
+app = Flask(__name__)
+api = Api(app)
+CORS(app)
```

Comment:
Need to look into handling CORS problems when using a locally hosted web application in a browser to access a local socket process